dwave/system/temperatures.py

```diff
@@ -249,373 +384,685 @@ def maximum_pseudolikelihood_temperature(bqm=None,
 
         https://www.jstor.org/stable/25464568
 
-    '''
-
-    T_estimate = 0
-    T_bootstrap_estimates = np.zeros(num_bootstrap_samples)
-
-    #Check for largest local excitation in every sample, and over all samples
-    if site_energy is None:
-        if bqm is None or sampleset is None:
-            raise ValueError('site_energy can only be derived if both'
-                             'bqm and sampleset are provided as arguments')
-        site_energy = effective_field(bqm,
-                                      sampleset,
-                                      current_state_energy = True)
-    max_excitation = np.max(site_energy[0],axis=1)
-    max_excitation_all =  np.max(max_excitation)
-    if max_excitation_all <= 0:
-        #There are no local excitations present in the sample set, therefore
-        #the temperature is estimated as 0. 
-        pass
+    """
+    x0 = None
+    bisect_bracket = None
+    if T_guess is not None:
```

Comment:
(minor) but would also avoid the unfortunate case where `T_guess == 0`
```suggestion
    if T_guess:
```